packages/actions/src/helpers/storage.ts

```diff
@@ -80,18 +80,23 @@ export const getChunksAndPreSignedUrls = async (
  * @param cloudFunctions <Functions> - the Firebase Cloud Functions service instance.
  * @param ceremonyId <string> - the unique identifier of the ceremony.
  * @param alreadyUploadedChunks Array<ETagWithPartNumber> - the temporary information about the already uploaded chunks.
+ * @param logger <any> - an optional logger to show progress.
  * @returns <Promise<Array<ETagWithPartNumber>>> - the completed (uploaded) chunks information.
  */
 export const uploadParts = async (
     chunksWithUrls: Array<ChunkWithUrl>,
     contentType: string | false,
     cloudFunctions?: Functions,
     ceremonyId?: string,
-    alreadyUploadedChunks?: Array<ETagWithPartNumber>
+    alreadyUploadedChunks?: Array<ETagWithPartNumber>,
+    logger?: any
```

Comment:
Is this PR related to the quick UI fix for zkp2p setup you mentioned earlier? I noticed the `logger` argument is of type `any`, and we're assuming it will have `start()` and `increment()` methods. This might be prone to errors. What do you think?

Reply:
I added the type definition